Exiled.Events/Patches/Fixes/AntiCheatSpeedFix.cs

```diff
@@ -7,127 +7,69 @@
 
 namespace Exiled.Events.Patches.Fixes
 {
-#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
-
-    using System;
     using System.Collections.Generic;
-
-    using CustomPlayerEffects;
+    using System.Reflection.Emit;
 
     using Exiled.API.Features;
 
     using HarmonyLib;
 
-    using InventorySystem.Items;
-    using InventorySystem.Items.Armor;
+    using NorthwoodLib.Pools;
 
-    using PlayableScps;
-
-    using UnityEngine;
+    using static HarmonyLib.AccessTools;
 
     /// <summary>
-    /// Fixes AntiCheat making rollback the player when he get there speed changed"/>.
+    /// Fixes position desync when player speed changed"/>.
+    /// Adds <see cref="Player.RunningSpeed"/> and <see cref="Player.WalkingSpeed"/> implementation.
     /// </summary>
     [HarmonyPatch(typeof(FirstPersonController), nameof(FirstPersonController.GetSpeed))]
     public static class AntiCheatSpeedFix
```

Comment:
Should probably be made internal to meet the current transpiler style

Reply:
Done